include/zephyr/bluetooth/hci.h

```diff
@@ -1899,6 +1899,21 @@ struct bt_hci_rp_le_read_iso_tx_sync {
 	uint8_t  offset[3];
 } __packed;
 
+#define BT_HCI_CIG_ID_MAX                       0xFE
+#define BT_HCI_CIS_COUNT_MAX                    0x1F
+#define BT_HCI_SDU_INTERVAL_MIN                 0x0000FF
+#define BT_HCI_SDU_INTERVAL_MAX                 0x0FFFFF
+#define BT_HCI_WORST_CASE_SCA_VALID_MASK        0x07
+#define BT_HCI_PACKING_VALID_MASK               0x01
+#define BT_HCI_FRAMING_VALID_MASK               0x01
+#define BT_HCI_MAX_TRANSPORT_LATENCY_MIN        0x0005
+#define BT_HCI_MAX_TRANSPORT_LATENCY_MAX        0x0FA0
+#define BT_HCI_CIS_ID_VALID_MAX                 0xEF
+#define BT_HCI_MAX_SDU_VALID_MASK               0x0FFF
+#define BT_HCI_PHY_VALID_MASK                   0x07
```

Comment:
Suggest to prefix all these with `BT_HCI_ISO` as something like `BT_HCI_SDU_INTERVAL_MIN` is too generic and could apply to non-ISO SDUs

Reply:
Perhaps with the exceptions with values prefixed with CIS/CIG/BIS/BIG

Reply:
I guess since these are defined by the HCI spec, we don't need to add documentation nor add a unit suffix?